package.json

```diff
@@ -65,24 +64,36 @@
     "@storybook/addon-essentials": "7.6.17",
     "@storybook/addon-links": "7.6.17",
     "@storybook/addon-styling-webpack": "^0.0.6",
+    "@storybook/blocks": "^7.0.27",
+    "@storybook/components": "^7.0.27",
+    "@storybook/core-events": "^7.0.27",
+    "@storybook/preview-api": "^7.0.27",
+    "@storybook/theming": "^7.0.27",
+    "@storybook/types": "^7.0.27",
     "@storybook/vue": "7.6.17",
     "@storybook/vue-webpack5": "7.6.17",
+    "@storybook/vue3": "^7.0.27",
```

Comment:
Why is this here?

Reply:
Actually I wonder too. But yarn pointed out, that was an unresolved dependency. Since storybook is sort of standalone, I can imagine, that storybook 7 uses vue3. Maybe we don't need @storybook/vue... let me find out....

Reply:
storybook 8 uses vue3, afaics.

Reply:
> storybook 8 uses vue3, afaics.

For me it looks like SB 7.6.17 already runs with vue3. at least in some sort of compat mode.